packages/server/modules/core/graph/schemas/streams.graphql

```diff
@@ -70,7 +70,7 @@ type StreamCollaborator {
 type StreamCollection {
   totalCount: Int!
   cursor: String
-  items: [Stream]
+  items: [Stream!]
```

Comment:
doesn't make sense that Stream was nullable in the first place (do we really allow `[null]` arrays?), do you guys predict any issues with this? @didimitrie @gjedlicska

Reply:
IIRC, the intention was to allow for empty arrays `[]` - that's all. If the new way allows for that, we're cool

Reply:
yea, it does, should be safe.

fyi:
`[X!]!` -> `[]` or `[X]`
`[X!]` -> `[]` or `[X]` or `null`
`[X]` -> `[]` or `null` or `[X]` or `[X, null]`